code/modules/error_handler/error_handler.dm

```diff
@@ -26,7 +26,9 @@ GLOBAL_VAR_INIT(total_runtimes_skipped, 0)
 		Reboot(reason = 1)
 		return
 
-	var/static/regex/stack_workaround = regex("[WORKAROUND_IDENTIFIER](.+?)[WORKAROUND_IDENTIFIER]")
+	var/static/regex/stack_workaround
+	if(isnull(stack_workaround))
```

Comment:
This should not be making any functional difference. Are you saying all static variables in the entire code base should be initialized this way?Why does this work?